python/tools/desktop_onboarding.py

```
@@ -0,0 +1,291 @@
+application later.
+"""
+from __future__ import annotations
+
+import argparse
+import dataclasses
+import os
+import platform
+import shutil
+import subprocess
+import sys
+from typing import Iterable, Optional
```

Ruff (0.14.0)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)